manifest.json

```diff
@@ -10,8 +10,7 @@
     "default_popup": "popup.html"
   },
   "permissions": [
-    "activeTab",
-    "storage"
```

Comment:
why was storage removed? are you not storing the jwt in localstorage?

Reply:
thats beyond the scope of hooking the extension to the auth API (#16)
The storage of the JWT is in the scope of #19 
I can make a PR after for with that enhancement

Reply:
yeah but wouldn't I have to login every time I click on the chrome extension? the two tasks are hand-in-hand

Reply:
yes, you currently will have to login every time you click on the extension.
this PR only addresses login in and out for the `Login` components perspective.
#19 handles login in and out from a broader perspective.

this PR ensures that first step is functional in order to work on the next PRs (#17 #19)

Reply:
I vote against holding up the PR until #19 is done.
storing the JWT in storage has nothing to do with the `Login` component and its call to the Auth API